ntbs-service/Models/Enums/Status.cs

```diff
@@ -1,8 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
 namespace ntbs_service.Models.Enums 
 {
     public enum Status {
         Yes,
         No,
         Unknown
     }
+
+    public static class StatusHelper

```

Comment:
This was previously defined as a model property in socialriskfactors, but is also needed by comorbidities. I think it makes more sense in a helper!